xontrib/vox.py

```diff
@@ -1,23 +1,11 @@
-"""Python virtual environment manager for xonsh."""
-import os
-import sys
-import venv
-import shutil
-import builtins
-
-from xonsh.platform import ON_POSIX, ON_WINDOWS, scandir
-
-
-class Vox:
+class _VoxHandler:
     """Vox is a virtual environment manager for xonsh."""
 
     def __init__(self):
         """Ensure that $VIRTUALENV_HOME is defined and declare the available vox commands"""
 
-        if not builtins.__xonsh_env__.get('VIRTUALENV_HOME'):
-            home_path = os.path.expanduser('~')
-            venvdir = os.path.join(home_path, '.virtualenvs')
-            builtins.__xonsh_env__['VIRTUALENV_HOME'] = venvdir
+        import xontrib.voxapi
```

Comment:
I think this is better as `import xontrib.voxapi as _voxapi` globally.

The reason the mpl hooks did imported lazily is because matplotlib and numpy take a long time to import.  That shouldn't be a problem here.